package.json

```diff
@@ -13,9 +13,9 @@
   "readmeFilename": "README.md",
   "gitHead": "eb6f38decdf15fc18625bf60a602db2937feb229",
   "scripts": {
-    "build": "$(npm bin)/grunt",
-    "release": "$(npm bin)/grunt release",
-    "watch": "$(npm bin)/grunt watch"
```

Comment:
I guess I overlooked the fact it wouldn't work cross-platform, but I didn't want globally installing grunt to be a dependency. I wonder if there is a cross-platform way to achieve that... To the internets!